libraries/adsk/shaders/wood/parquet_3/parquet_03.mtlx

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+  <nodedef name="ND_parquet_03_shader" node="parquet_03" nodegroup="Shaders">>
```

Comment:
Example shader definition. Made up nodegroup of "Shader". A material can use this directly or we can have mateiral definition or presets using shader definitions (e.g. Substance exports shader definitions + preset).